pkg/controller/metrics/custom_collectors.go

```diff
@@ -54,7 +54,7 @@ func (cc provisioningUnderwayCollector) Collect(ch chan<- prometheus.Metric) {
 	clusterDeployments := &hivev1.ClusterDeploymentList{}
 	err := cc.client.List(context.Background(), clusterDeployments)
 	if err != nil {
-		log.WithError(err).Error("error listing cluster deployments")
+		ccLog.WithError(err).Error("error listing cluster deployments")
```

Comment:
We were initializing ccLog but using the one without the field set to metrics controller - corrected this in the entire file